src/main/java/uk/gov/justice/probation/courtcaseservice/jpa/entity/GroupedOffenderMatchesEntity.java

```diff
@@ -0,0 +1,55 @@
+package uk.gov.justice.probation.courtcaseservice.jpa.entity;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import io.swagger.annotations.ApiModel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.RequiredArgsConstructor;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinColumns;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+import java.io.Serializable;
+import java.util.List;
+
+@ApiModel(description = "Grouped Offender Matches")
+@Entity
+@AllArgsConstructor
+@RequiredArgsConstructor
+@Builder
+@Data
+@Table(name = "OFFENDER_MATCH_GROUP")
+public class GroupedOffenderMatchesEntity  implements Serializable {
+    @Id
+    @Column(name = "ID", updatable = false, nullable = false)
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @JsonIgnore
+    private Long id;
+
+    @OneToMany(fetch = FetchType.EAGER, cascade = { CascadeType.ALL }, orphanRemoval=true)
+    private List<OffenderMatchEntity> offenderMatches;
+
+    @ManyToOne(cascade = CascadeType.ALL, fetch = FetchType.LAZY)
+    @JoinColumns({
+            @JoinColumn(name = "CASE_NO", referencedColumnName = "case_no", nullable = false),
+            @JoinColumn(name = "COURT_CODE", referencedColumnName = "court_code", nullable = false),
+    })
+    @JsonIgnore
+    private CourtCaseEntity courtCaseEntity;
+
+    @Column(name = "CASE_NO", nullable = false, insertable = false, updatable = false)
```

Comment:
> Don't think that caseNo and courtCode need / shoud be here. The exist on the court case entity. Same sort of relationship as between OffenceEntity and CourtCaseEntity.

I included these so that we didn't have to bring back the entity in order to get these fields but ok I can remove them, just seemed more convenient.

> As mentioned in the SQL ca the CourtCaseEntity for the group ever be different on any of the individual offenderMatches ? If so, let's not duplicate the relationship

Done.